File: java/exer2/kart/Motor.java
package java.exer2.kart;

public class Motor {
  private String cilindradas;
  private float velocidadeMaxima;

  public Motor(String cilindradas, float velocidadeMaxima) {
    this.cilindradas = cilindradas;
    this.velocidadeMaxima = velocidadeMaxima;
  }

  public void mostraInfo() {
    System.out.println("O motor tem " + cilindradas + "cilindradas e a velocidade max e " + velocidadeMaxima + "Km/h.");
  }

}

File: python/kart/motor.py
class Motor:
    def __init__(self, cilindradas, velocidadeMaxima):
        self.cilindradas = cilindradas
        self.velocidadeMaxima = velocidadeMaxima

    def mostraInfo(self):
        print(
            f"O motor tem {self.cilindradas} cilindradas e a velocidade máxima é {self.velocidadeMaxima} Km/h."
        )
